serverless/database-backup/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "database-backup",
+  "version": "1.0.0",
+  "description": "Backup database to S3",
+  "main": "src/index.js",
+  "scripts": {
+    "build:docker": "docker run --rm  -e PG_VERSION=$PG_VERSION -v $PWD:/var/task lambci/lambda:build-nodejs12.x ./build.sh",
```

Comment:
why did you choose to run a build script directly, instead of building a dockerfile from the lambci/lambda:build-nodejs12.x image?  
where is PG_VERSION set?

Reply:
`PG_VERSION` is set in Travis. Not too sure if that's the best place to track this actually. The [docs](https://www.postgresql.org/docs/11/app-pgdump.html#PG-DUMP-NOTES) mentioned that we should use the same major version as the server you're dumping from. I've added this instead of hardcoding some value to allow us to update without code change when we upgrade to a new major version of Postgres.

I was thinking that if I use a Dockerfile, I'll have to do the following to get the built lambda zip out since you can't do volume mounts for `docker build`. 

```sh
$ docker build -t database-backup -f Dockerfile .
$ docker run --rm -v $PWD:/output database-backup cp /var/task/code.zip /output/code.zip
```

Doesn't seem to be a way to copy files out of a built image without running the container. 

Chose to directly run the script instead since it can be done in one step. Downside is that it might pollute your mounted host directory or a bug in the script might accidentally rm some stuff. But given that this script is only run in the CI server, my thoughts are that it's ok. Open to changing.